src/main/resources/application.yaml

```diff
@@ -22,3 +19,9 @@ app-insights:
 spring:
   application:
     name: Send Letter Provider
+
+servicebus:
+  connectionString: ${SERVICE_BUS_CONNECTION_STRING}
+  queue:
+    name: ${SERVICE_BUS_QUEUE_NAME}
```

Comment:
one more thing, could we have these required environment vars in the `run-in-dokcer` script so it stands for "bare minimum to successfully run in docker" statement?

Reply:
I thought I will fix all docker stuff as a separate PR but will fix this now.

Reply:
oh, didn't know. nw, can be separate